buildenv/jenkins/JenkinsfileBase

```diff
@@ -26,7 +26,6 @@ def stageTestBinaries() {
 }
 
 def setupEnv() {
-	env.JCL_VERSION = "current"
```

Comment:
Could also remove this Env in personal jenkins file?

Reply:
Sure, I can update the personal. I think we need to revisit personal jenkins file. It needs to use the functions in base.

Reply:
I have a PR open that has some changes to the personal.  If you are working on that file, perhaps take a look at my "grinder" branch to see what other items are coming soon.